src/teraranger_evo.cpp

```diff
@@ -0,0 +1,215 @@
+#include <ros/console.h>
+#include <string>
+
+#include <teraranger_hub/RangeArray.h>
```

Comment:
Also, in case of Multiflex and One arrays we are not publishing range array. I propose that we make sure that all 3 nodes publish messages in the same way otherwise it might become very difficult to support.